server/src/providers/about-transactions.ts

```diff
@@ -0,0 +1,123 @@
+import AboutTransactionsModel from '../models/about-transactions'
+import logger from '../helpers/pino-logger'
+
+const log = logger.child({ from: 'Transactions' })
+
+type aboutTransactions = {
+  count_txs: number,
+  amount_txs: number,
+  date: string,
+}
+
+class AboutTransactions {
+
+  model:any
+
+  constructor() {
+    this.model = AboutTransactionsModel
+  }
+
+  /**
+   * Create or update wallet
+   *
+   * @param {transaction} transaction
+   *
+   * @returns {Promise<void>}
+   */
+
+
+  async set(aboutTransactions: aboutTransactions): Promise<boolean> {
+    try {
+      await this.model.create(aboutTransactions)
+      return true
+    } catch (ex) {
+      log.error('Update user failed [mongo actions]:', { message: ex.message, aboutTransactions})
+    }
+
+    return false
+  }
+
+  async updateOrSet(aboutTransactions: aboutTransactions): Promise<boolean> {
+    try {
+      const aboutTXs:any = await this.getByDate(aboutTransactions.date)
+      if (aboutTXs) {
+        aboutTXs.count_txs = aboutTXs.count_txs+aboutTransactions.count_txs
```

Comment:
use $inc.
one operation for update!

Reply:
we need to check first if `date` in aboutTransactions already exists. so we get the model. so that's why we already have the model so we don't need to use $inc we need just use model.save() function. @sirpy

Reply:
use $inc. for save txInfo and wallet